DYN-7833 Update TuneUp

QilongTang opened the pull request:
### Purpose

Update TuneUp in Dynamo built-in package per https://github.com/DynamoDS/TuneUp/pull/86

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

When switching to custom node workspace, `Run All` button on TuneUp will be disabled

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

reddyashish commented:
@QilongTang pkg.json under the tuneup should be updated too. Is this https://github.com/DynamoDS/TuneUp/pull/86 going to be the only fix for 2.0.1?

QilongTang commented:
> @QilongTang pkg.json under the tuneup should be updated too. Is this [DynamoDS/TuneUp#86](https://github.com/DynamoDS/TuneUp/pull/86) going to be the only fix for 2.0.1?

Yes, the only fix and done addressing comment.
